src/02-internuclear-potentials.md

```diff
@@ -0,0 +1,265 @@
+
+
+The electrostatic potential energy $U$ (in J) between two charged particles is given by:
+
+$$U = \frac{q_1 q_2}{4\pi\varepsilon_0 r}$$
+
+where:
+
+$q$ is the charge in C,
+
+$r$ is the distance between the charges in m.
+
+It can be convenient to work with charge in units of the [elementary electric charge](https://en.wikipedia.org/wiki/Elementary_charge) $e = 1.6\times 10^{-19}$ C. In this case the energy can be written as
+
+$$U = \frac{Z_1 Z_2 e^2}{4\pi\varepsilon_0 r}$$
+
+where
+
+$Z$ is the [charge number](https://en.wikipedia.org/wiki/Charge_number).
+
+It can also be convenient to calculate this energy in [electron volts](https://en.m.wikipedia.org/wiki/Electronvolt) (eV). 1eV is the energy acquired by an electron after being accelerated through a voltage of 1V. Hence 
+
+$1\text{eV} = e\text{J} = 1.602176634\times 10^{-19} \text{J}$
+
+and therefore
+
+$$U_{eV} = \frac{Z_1 Z_2 e}{4\pi\varepsilon_0 r}$$
+
+Finally, if we work in [Hartree atomic units (a.u.)](https://en.wikipedia.org/wiki/Hartree_atomic_units), then this expression simplifies to:
+
+$$U_{au} = \frac{Z_1 Z_2}{r}$$
+
+
+```python
+# when working in au then both e and the denominator except r are 1
+def coulomb_au(Z_1,Z_2,r):
+  U = Z_1*Z_2 / r
+  return U
+```
+
+```python
+Rpm2 = np.arange(1,100,0.1)
+
+Cev =  coulomb_au(1,1,pm_to_bohr(Rpm2)) - 1.9037 # subtract a constant according to K&N
+```
+
+```python
+hartree_to_ev(1)
+```
+
+```python
+plt.plot(kw_copy["Rpm"],kw_copy["Eev"]+hartree_to_ev(0.65),label="KW potential")
+plt.plot(Rpm2,hartree_to_ev(Cev),label="Coulomb potential only")
+
+plt.ylim(-40,100)
+plt.xlim(-0,100)
+plt.xlabel("internuclear distance [pm]")
+plt.ylabel("potential energy [eV]")
+plt.title("Coulomb potential and KW potential")
+plt.legend()
+plt.show()
+```
+
+## Adding a Woods-Saxon nuclear potential
+
+
+At close range on the order of a few fm the repulsion of the Coulomb potential gets overridden by the attraction of nucleons by the nuclear force. This potential is often modeled through the [Woods-Saxon potential](https://en.wikipedia.org/wiki/Woods%E2%80%93Saxon_potential):
+
+$$V(r)=-{\frac  {V_{0}}{1+\exp({r-R \over a})}}$$
+
+where 
+
+$$R=r_{0}A^{{1/3}}$$
+
+$r_0$, $V_0$ and a are fitted constants; $r$ is the radius from the center of the nucleus; and $A$ is the mass number of the nucleus whose potential is expressed (i.e. the number of nucleons).
+
+The Woods-Saxon (WS) potential is a mean potential that is assumed to reasonably describe the attraction of the nuclear force from a given nucleus with A=number of nucleons.
+
+```python
+A = 4 # 4 nucleons in two deuterium atoms
+V0 = 28.5e6 #50 # ground state energy see for He-4 https://pdfs.semanticscholar.org/4c24/b2d33f3b3ca2f288897967dc3a6fdb51f1a0.pdf
```

Comment:
A few comments here
- I see why you chose 28.5, but I don't think it is as simple as putting in the ground state energy - see http://www.uprh.edu/rbaretti/WoodsSaxonPotential25may2012.htm
- The link you reference, the paper seems to be suggesting a novel way to look at nuclear levels including gravity. Is there are more mainstream reference we can use?
- In general I think we are mixing up potential and potential energy. Morse, KW and coulomb seem to be potential energy, but WS is potential.